src/ui/EditorNS/editor.cpp

```diff
@@ -125,6 +134,20 @@ namespace EditorNS
         return out;
     }
 
+    void Editor::on_loadFinished()
+    {
+        updateBackground();
+    }
+
+    void Editor::updateBackground()
+    {
+        QString js = "rgb2hex($('.cm-s-'+editor.getOption('theme')).css('background-color'));";
+        m_webView->page()->runJavaScript(js, [&] (const QVariant& v){
```

Comment:
Probably a good idea... might create a struct that pulls all the theme data, if you think it would be useful... otherwise we'll just turn the background portion into its own function.